util/FetchConfig.ts

```diff
@@ -1,24 +1,24 @@
 /**
  * Fetches config from environment and API.
- * @returns A promise to get & set session storage keys with appropriate values.
+ * @return A promise to get & set session storage keys with appropriate values.
  */
 export default async function fetchConfig (): Promise<any> {
-    // get config from environment
-    const origin: string = window.location.origin
-    const windowLocationPathname: string = window.location.pathname // eg, /basePath/...
-    const windowLocationOrigin: string = window.location.origin // eg, http://localhost:8080
-    const windowLocationSearch: string = window.location.search
-  
-    if (!origin || !windowLocationPathname || !windowLocationOrigin) {
-      return Promise.reject(new Error('Missing environment variables'))
-    }
- 
-    const keycloakAuthUrl: string = process.env.VUE_APP_KEYCLOAK_AUTH_URL || 'default-url';
-    (<any>window).keycloakAuthUrl = keycloakAuthUrl
-  
-    const keycloakRealm: string = process.env.VUE_APP_KEYCLOAK_REALM|| 'default-realm';
-    (<any>window).keycloakRealm = keycloakRealm
-  
-    const keycloakClientId: string = process.env.VUE_APP_KEYCLOAK_CLIENTID|| 'default-id';
-    (<any>window).keycloakClientId = keycloakClientId
-  }
\ No newline at end of file
+  // get config from environment
+  const origin: string = window.location.origin
+  const windowLocationPathname: string = window.location.pathname // eg, /basePath/...
+  const windowLocationOrigin: string = window.location.origin // eg, http://localhost:8080
+  const windowLocationSearch: string = window.location.search
+
+  if (!origin || !windowLocationPathname || !windowLocationOrigin) {
+    return Promise.reject(new Error('Missing environment variables'))
+  }
+
+  const keycloakAuthUrl: string = process.env.VUE_APP_KEYCLOAK_AUTH_URL || 'default-url';
+  (<any>window).keycloakAuthUrl = keycloakAuthUrl
+
+  const keycloakRealm: string = process.env.VUE_APP_KEYCLOAK_REALM || 'default-realm';
+  (<any>window).keycloakRealm = keycloakRealm
+
+  const keycloakClientId: string = process.env.VUE_APP_KEYCLOAK_CLIENTID || 'default-id';
+  (<any>window).keycloakClientId = keycloakClientId
```

Comment:
For the record, this is one of 3 ways that the app stores temporary data:
1. add to `window` object (what's done here)
2. add to session storage
3. add to store

As this feature is a re-write of the old project, this is an opportunity to reduce the ways of storing temp data. Thoughts?

Reply:
I'm thinking of storing temporary data in either session storage or the store exclusively. Do you think this approach would be beneficial?

Reply:
I think having fewer ways to store temporary data would be beneficial, yes.